class-openid-connect-server.php

```diff
@@ -221,15 +221,18 @@ public function jwks() {
 		header( 'Content-type: application/json' );
 		header( 'Access-Control-Allow-Origin: *' );
 
-		$options = array(
+		$keySet = new \Strobotti\JWK\KeySet();
+		$keyFactory = new \Strobotti\JWK\KeyFactory();
```

Comment:
Could we import the code from the library? I think it's just a few openssl function calls, then we could drop the dependency.